homeassistant/components/switchbot/config_flow.py

```diff
@@ -144,6 +153,38 @@ async def async_step_password(
             },
         )
 
+    async def async_step_lock_key(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle the encryption key step."""
+        errors = {}
+        assert self._discovered_adv is not None
+        if user_input is not None:
+            if not await SwitchbotConfigFlow._verify_lock_configuration(
```

Comment:
Lets not make this a bound method since it doesn't use `self`.  Ideally this would actually be in the library instead